pimd/pimd.h

```diff
@@ -157,23 +157,21 @@ extern uint8_t qpim_ecmp_rebalance_enable;
 	(router->debugs & PIM_MASK_PIM_PACKETDUMP_SEND)
 #define PIM_DEBUG_PIM_PACKETDUMP_RECV                                          \
 	(router->debugs & PIM_MASK_PIM_PACKETDUMP_RECV)
-#define PIM_DEBUG_PIM_TRACE (router->debugs & PIM_MASK_PIM_TRACE)
+#define PIM_DEBUG_PIM_TRACE                                                    \
+	(router->debugs & (PIM_MASK_PIM_TRACE | PIM_MASK_PIM_TRACE))
```

Comment:
Why is PIM_MASK_PIM_TRACE mentioned twice here?

Reply:
Haha, oops, not enough coffee error… will fix in a minute